craft_providers/lxd/launcher.py

```diff
@@ -32,6 +89,8 @@ def launch(
     auto_clean: bool = False,
     ephemeral: bool = False,
     map_user_uid: bool = False,
+    use_snapshots: bool = False,
+    lxc: LXC = LXC(),
```

Comment:
Please document also this parameter below, thanks!

Reply:
Ah yes, will do thnx